apps/webstore/src/app/shared/components/modals/family-tree-template-modal/family-tree-template-modal.component.scss

```diff
@@ -33,31 +29,29 @@
   justify-content: center;
 }
 
-::ng-deep .carousel-indicators li {
-  background-color: var(--second-action-color);
-  opacity: 0.3;
-}
-
-::ng-deep .carousel-indicators {
-  margin-bottom: 0;
+.template-container {
+  display: flex;
+  min-width: 350px;
+  margin: 2em 4em;
+  overflow-x: scroll;
+  text-align: center;
+  padding-bottom: 10px;
 }
 
-::ng-deep .carousel-control-prev-icon {
-  background-image: url('/assets/icons/keyboard-arrow-left.svg');
-  top: 18px;
-  position: relative;
+/* width */
+::-webkit-scrollbar {
```

Comment:
It will not work on Safari on iPhone, and will only partially work on safari and chrome on Mac
@Teodor25 I've mentioned to you before regarding the home page review section. We can't reliably change the styling of the scroll bar